amclient/amclientargs.py

```diff
@@ -132,6 +133,27 @@
     default="amclient-transfer",
     type=None,
 )
+MICROSERVICE_GROUP = Opt(
+    name="microservice-group",
+    metavar="MICROSERVICEGROUP",
+    help="Filter by microservice group",
+    default=None,
+    type=None,
+)
+MICROSERVICE_LINK_UUID = Opt(
+    name="microservice-link-uuid",
+    metavar="MICROSERVICELINKUUID",
+    help="Filter by microservice link UUID",
+    default=None,
+    type=None,
+)
+MICROSERVICE_TYPE = Opt(
+    name="microservice-type",
+    metavar="MICROSERVICETYPE",
+    help="Filter by microservice type",
```

Comment:
I found this a little confusing to decode. Would it make sense to rename this `microservice-job-name`?

Reply:
Good point. After asking the team maybe we could change `microservice` to `job` in the new endpoints and amclient code (`get-microservices` -> `get-jobs`, etc). So we could make this filter option `JOB_NAME` and so on. What do you think?

Reply:
Sounds good!